<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export As</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBoxPRG">
     <property name="title">
      <string>Location</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>editFilename</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editFilename"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushBrowse">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>What to export</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox_charset">
        <property name="text">
         <string>Charset</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_map">
        <property name="text">
         <string>Map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_tileColors">
        <property name="text">
         <string>Tile Colors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gridGroupBox">
     <property name="title">
      <string>File Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_asm">
        <property name="text">
         <string>ASM</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Map Addr.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_raw">
        <property name="text">
         <string>Raw Format</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBox_charsetAddress">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="prefix">
         <string>$</string>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="value">
         <number>14336</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinBox_mapAddress">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="prefix">
         <string>$</string>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="value">
         <number>16384</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Tile Colors Addr.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_c">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="radioButton_prg">
        <property name="text">
         <string>PRG Format</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spinBox_attribAddress">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="prefix">
         <string>$</string>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="value">
         <number>17408</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Charset Addr.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButton_png">
        <property name="text">
         <string>PNG</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editFilename</tabstop>
  <tabstop>pushBrowse</tabstop>
  <tabstop>checkBox_charset</tabstop>
  <tabstop>checkBox_map</tabstop>
  <tabstop>checkBox_tileColors</tabstop>
  <tabstop>radioButton_asm</tabstop>
  <tabstop>radioButton_raw</tabstop>
  <tabstop>radioButton_prg</tabstop>
  <tabstop>spinBox_charsetAddress</tabstop>
  <tabstop>spinBox_mapAddress</tabstop>
  <tabstop>spinBox_attribAddress</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
